File: projectBlueWater/src/main/java/com/lasthopesoftware/bluewater/client/playback/service/broadcasters/IPlaybackBroadcaster.java
package com.lasthopesoftware.bluewater.client.playback.service.broadcasters;

import com.lasthopesoftware.bluewater.client.playback.file.PositionedFile;

public interface IPlaybackBroadcaster {
    void sendPlaybackBroadcast(String broadcastMessage, int libraryId, PositionedFile positionedFile);
}

File: projectBlueWater/src/main/java/com/lasthopesoftware/bluewater/client/playback/service/broadcasters/IPlaybackBroadcaster.kt
package com.lasthopesoftware.bluewater.client.playback.service.broadcasters

import com.lasthopesoftware.bluewater.client.browsing.library.repository.LibraryId
import com.lasthopesoftware.bluewater.client.playback.file.PositionedFile

interface IPlaybackBroadcaster {
	fun sendPlaybackBroadcast(broadcastMessage: String, libraryId: LibraryId, positionedFile: PositionedFile)
}
